### PR TITLE
added lint checker in CI and fixes lint errors

### LINKED ISSUE #184: Add Golang linting in CI and remove unreachable code from the repo
### Describe the issue you're facing

- There are few unused funcs/packages that need to be removed. Run Golang lint to check other lint errors.
- Add Golang lint job to CI, use https://golangci-lint.run/ or similar

### PR DESCRIPTION
**Pull request by Omkar0114**
### What does this PR change?

- add lint checker in CI and fixes the lint errors

### Does the PR depend on any other PRs or Issues? If yes, please list them.

- fixes #184 

### Checklist

I confirm, that I have...

- [X] Read and followed the contributing guide in `CONTRIBUTING.md`
- [ ] Added tests for this PR
- [ ] Formatted the code using `go fmt` (if applicable)
- [ ] Updated [documentation on the Paralus docs site](https://github.com/paralus/website/blob/main/docs) (if applicable)
- [ ] Updated `CHANGELOG.md`

### CODE REVIEW

File: server/relaypeerservice.go [hunk 1]
```diff
@@ -156,19 +156,19 @@ func (s *relayPeerService) handleSurveyReq(req surveyBroadCastRequest) {
 
 	foundStale = false
 
-	//broadcast request to all connected relays
+
 	s.relayMutex.RLock()
 	now := time.Now().Unix()
 	if relayList, ok := s.RelayMap[req.ou]; ok {
 		for relayuuid, robj := range relayList {
-			if now > robj.timeStamp && (now-robj.timeStamp) > int64(maxRelayIdle) { //5min max toleration
```
Comment (akshay196) [inline server/relaypeerservice.go:164]: Looks like this is removed by mistake. @Omkar0114  Can you check and revert back change?

Comment (akshay196) [reply server/relaypeerservice.go:164]: Also check below errors:
```
Error: server/relaypeerservice.go:186:2: expected declaration, found s
Error: server/relaypeerservice.go:406:2: expected declaration, found s
Error: server/relaypeerservice.go:451:2: expected declaration, found ctx
Error: server/relaypeerservice.go:537:2: expected declaration, found ctx
```

Comment (Omkar0114) [reply server/relaypeerservice.go:164]: I make the change revert in `server/relaypeerservice.go` and few errors I saw in the local copy. Let's check now

**Comment by akshay196** [comment]
@Omkar0114 There are more go lint errors. Can you please take a look and fix them?
You can run golangci-lint locally to confirm after errors are resolved.

**Comment by akshay196** [comment]
Hey @Omkar0114 Did you get a chance to take a look? Need to fix CI failures. Also rebase your branch.

**Comment by akshay196** [comment]
Hey @Omkar0114  Checking if you're still working on this PR.

**Comment by Omkar0114** [comment]
Hey @akshay196 
So sorry for the late reply as I was a little busy with my college stuff. I am not working on this issue right now.

**Comment by akshay196** [comment]
@Omkar0114 no worries. Unassigning issue so others can contribute.

**Comment by akshay196** [comment]
Closing because stale PR.

**Pull request closed without merging**